docs/guide/bundles.md

```diff
@@ -24,59 +24,130 @@ To learn more about organizing your eZ Platform project, see [Best Practices](be
+|[ez-support-tools](https://github.com/ezsystems/ez-support-tools)|provides functionality for system information|
+|[ezplatform-http-cache](https://github.com/ezsystems/ezplatform-http-cache)|HTTP cache handling for eZ Platform, using multi tagging (incl Varnish xkey)|
+|[ezplatform-design-engine](https://github.com/ezsystems/ezplatform-design-engine)|design fallback system for eZ Platform similar to legacy design fallback system|
+|[ezplatform-standard-design](https://github.com/ezsystems/ezplatform-standard-design)|defines standard Design and Theme to be handled by ezplatform-design-engine|
+|[ezplatform-cron](https://github.com/ezsystems/ezplatform-cron)|exposes cron/cron package for use in eZ Platform (or just plain Symfony) via a simple command `ezplatform:cron:run`|
+|[BehatBundle](https://github.com/ezsystems/BehatBundle)|common reusable sentence implementations and other common needs for Behat testing in eZ bundles/projects|
+
+!!! Enterprise
+
+    |Bundle|Description|
+    |---------|-----------|
+    |[ezplatform-ee](https://github.com/ezsystems/ezplatform-ee)|fork of the "ezplatform" meta repository, contains changes to composer.json that pull in all dependencies from updates.ez.no for eZ Platform Enterprise Edition (a commercial distribution of eZ Platform with additional features)|
+    |[ezpublish-kernel-ee](https://github.com/ezsystems/ezpublish-kernel-ee)|kernel for Enterprise, difference is the stable branches, master is automatically synced|
+    |[date-based-publisher](https://github.com/ezsystems/date-based-publisher)|provides the date based publishing functionality for the eZ Studio product|
+    |[flex-workflow](https://github.com/ezsystems/flex-workflow)|implementation of a collaboration feature that lets you send content draft to any user for a review or rewriting|
+    |[ezplatform-http-cache-fastly](https://github.com/ezsystems/ezplatform-http-cache-fastly)|extends ezplatform-http-cache to support Fastly, for use on Platform.sh PE or standalone|
+
+### Platform usage
+
+|Bundle|Description|
+|---------|-----------|
+|[ez-js-rest-client](https://github.com/ezsystems/ez-js-rest-client)|JavaScript REST client for eZ Platform RESTv2 API, mimics PHP API found in eZ Platform|
+|[ezplatform-i18n](https://github.com/ezsystems/ezplatform-i18n)|centralized internationalization|
+|[launchpad](https://github.com/ezsystems/launchpad)|CLI tool to bootstrap an eZ Platform project Docker stack|
+|[ezplatform-multi-file-upload](https://github.com/ezsystems/ezplatform-multi-file-upload)|allows uploading multiple files as new content items at once|
+|[satis](https://github.com/ezsystems/satis)|simple static Composer repository generator|
+|[ezplatform-demo-assets](https://github.com/ezsystems/ezplatform-demo-assets)|contains binary install data for ezsystems/ezplatform-demo|
+|[requirements](https://github.com/ezsystems/requirements)|requirements repository for eZ Platform development|
```

Comment:
dead repo, please skip

AND please skip ANY non package repos in bundle listing, if they are not composer packages they are definitely not bundles.

Thew one below should be listed somehow, but it's not a bundle.